### ISSUE TITLE
EasyScore vs. VexTab?

### ISSUE THREAD

**Issue by evanrmurphy** [0xfe/vexflow#647]
Hi, I've been re-acquainting myself with VexFlow after a couple of years away. I'm now considering it for some notation rendering projects at SightReadingMastery. I'm glad to see that the VexFlow project is still alive and receiving active contributions.

A couple years ago VexTab looked like the go-to high-level language for creating scores with VexFlow. However, it now appears from GitHub that the VexTab project isn't very active, and the VexFlow documentation references EasyScore instead. (Although www.vexflow.com still features VexTab prominently.)

Am I right that EasyScore replaces VexTab for most purposes? Or do they solve different problems? Some clarification about the status/relationship of EasyScore and VexTab would be appreciated.

**Comment by sschmidTU** [comment]
The main and active version of VexFlow is this Javascript library here, it was also called Vexflow 2.
VexTab is inactive, [last significant commit 2 years ago](https://github.com/0xfe/vextab/commits/master).
EasyScore does not completely cover VexFlow yet, [last commit also 2 years ago](https://github.com/0xfe/vexflow/blob/master/src/easyscore.js).

So, it's unfortunate that the easiest to use versions of VexFlow aren't up to date, and you have to program in Javascript for all features, but i think EasyScore can still do a lot.
I liked the simplicity and conciseness of VexTab.

You can use most of VexFlow's capabilities with MusicXML now, using [OSMD](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay).
Sorry for the advertising!

**Comment by acidjunk** [comment]
I'm also interested in knowing the best format to store music notation. I'm using OSMD for rendering bigger parts and it doet the job very well. But I also have some parts where I want to use some more exotique vexflow functionality that just isn't covered by MusicXML. Right now I'm storing the music in lilypond format and I convert it to MusicXML and/or a .json based format that I need to convert to EasyScore when I want to render it. I did find references in the issues ( for example #520 ) that other people also struggle with a nice, future proof, way to store complex notation for their needs, without loosing to much of vexflow's amazing capabilities.

I found this (quite old) project that tries to create a simple wrapper so EasyScore can be serialized from .json:
https://github.com/rubiety/vexflow-json

I think it would be a killer feature if we could load/save EasyScore to a .json based format.
